src/cmake/shared-sources.cmake

```diff
@@ -246,4 +246,12 @@ endif()
 include(${CMAKE_SOURCE_DIR}/src/platforms/${MZ_PLATFORM_NAME}/sources.cmake)
 include(${CMAKE_SOURCE_DIR}/src/cmake/sentry.cmake)
 
+add_dependencies(shared-sources translations)
+
 mz_add_clang_tidy(shared-sources)
+if(TARGET shared-sources_clang_tidy_report)
```

Comment:
Why is this condition required here?

Reply:
We do not generate that target i.e if there is no clang-tidy binary found, so the target may or may not exist.

Reply:
Sure but the dependencies are being added into this target, so if we don't build this the `add_depencies` also doesn't do anything.

Reply:
FWIW, if you don't even create it the `TARGET` keyword is prob unnecessary. It's a nit though.